SDCAlertView.podspec

```diff
@@ -10,6 +10,7 @@ Pod::Spec.new do |s|
   s.platform     =	:ios, '7.0'
   s.source       =	{ :git => 'https://github.com/sberrevoets/SDCAlertView.git', :tag => "v#{s.version}" }
   s.source_files =	'SDCAlertView/Source/*.{h,m}'
+  s.exclude_files = 'SDCAlertView/Source/SDCDemoViewController.{h,m}'
```

Comment:
The right fix is to instead move `SDCDemoViewController` out of the `Source` directory since it's not really the source of this project. You can then remove this line as the podspec will automatically exclude the demo view controller.